test/utils/int/storageRemote.ts

```diff
@@ -55,3 +55,26 @@ async function getStoredNotification(id: string): Promise<StoredNotification | u
         return note;
     }, id);
 }
+
```

Comment:
Worth keeping these around?

Reply:
I've kept them around as they could be useful in the future. I was thinking maybe once we have settings or something like that.

Reply:
I have removed them now per your comments